test/e2e/framework/util.go

```diff
@@ -5319,3 +5325,18 @@ func GetClusterZones(c clientset.Interface) (sets.String, error) {
 	}
 	return zones, nil
 }
+
+// WaitForNodeHasTaintOrNot waits for a taint to be added/removed from the node until timeout occurs, whichever comes first.
+func WaitForNodeHasTaintOrNot(c clientset.Interface, nodeName string, taint *v1.Taint, wantTrue bool, timeout time.Duration) error {
+	if err := wait.PollImmediate(Poll, timeout, func() (bool, error) {
+		has, err := NodeHasTaint(c, nodeName, taint)
+		if err != nil {
+			Logf("Failed to check taint %s on node %s or not", taint.ToString(), nodeName)
+			return false, nil
+		}
+		return has == wantTrue, nil
+	}); err != nil {
+		return fmt.Errorf("Expect node %v to have taint = %v within %v: %v", nodeName, wantTrue, timeout, err)
```

Comment:
errors should be lowercase, but i see the file does not follow this pattern.

Reply:
@neolit123 Yeap, I will update along with other reviewers' comments.